hapi-fhir-jpaserver-subscription/src/main/java/ca/uhn/fhir/jpa/subscription/module/subscriber/BaseSubscriptionDeliverySubscriber.java

```
@@ -71,9 +70,15 @@ public void handleMessage(Message theMessage) throws MessagingException {
 			myInterceptorBroadcaster.callHooks(Pointcut.SUBSCRIPTION_AFTER_DELIVERY, msg, msg.getSubscription());
 
 		} catch (Exception e) {
-			String msg = "Failure handling subscription payload for subscription: " + subscriptionId;
-			ourLog.error(msg, e);
-			throw new MessagingException(theMessage, msg, e);
+
+			// Interceptor call: SUBSCRIPTION_AFTER_DELIVERY
+			if (!myInterceptorBroadcaster.callHooks(Pointcut.SUBSCRIPTION_AFTER_DELIVERY_FAILED, msg, msg.getSubscription(), e)) {
+				return;
+			}
```

I wonder if the hook callback should happen after ourLog.error just in case someone doesn't know what they're doing and the exception gets swallowed...